### DISCUSSION TITLE
Running JBrowse 'forever' automatically...

### DISCUSSION THREAD

**Discussion by mickey-spongebob** [GMOD/jbrowse-components#3145] [Q&A] [answered]
Dear all,

Hope you're well! A stupid question perhaps, but I'd like to run my genome browser online 'forever' meaning that the server can continue running without me having to re-execute the 'npx serve . ' command. Is there something I can do to configure this? I apologise if this is too obvious...I'm new to setting up servers :-D

I'm currently running the server on an Ubuntu 20.04.4 LTS Virtual Machine.

Any advice would be greatly appreciated!
Thank you in advance,
an inexperienced kevin

**Comment by garrettjstevens** [comment, marked as answer]
Hi Kevin,

No problem, we all start somewhere. What you need is a web server like Apache (or nginx) to serve JBrowse. You should be able to find tutorials about how to set up Apache on whatever system you are on. Typically with this method you'll install JBrowse somewhere like `/var/www/html`, and the server will start automatically on boot and serve files from that location. This is the approach used in our [super-quick start guide](https://jbrowse.org/jb2/docs/superquickstart_web/). Let us know if you have any more questions.

**Comment by mickey-spongebob** [comment]
Awesome!

I did install everything correctly, just forgot to move it all to the /var/www/html/ folder (insert amateur here)…

Thank you very much and apologies once more for the trouble.

Best regards,
kevin
